$NetBSD: patch-ci,v 1.1 2002/10/30 16:57:03 kim Exp $

Fix detection of non-mime messages when the charset is not enclosed
in double quotes.

--- uip/show.c.orig	Sun Sep 12 09:50:12 1999
+++ uip/show.c	Wed Oct 30 11:42:42 2002
@@ -470,7 +470,7 @@
 				*bp = '\0';
 			} else {
 			    for (bp = dp; *bp; bp++)
-				if (isspace (*bp)) {
+				if (!istoken (*bp)) {
 				    *bp = '\0';
 				    break;
 				}
